.github/workflows/servo_example.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+name: Servo example
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      ANDROID_KEYSTORE_BASE64:
+        required: true
+      ANDROID_KEYSTORE_PASSWORD:
+        required: true
+
+jobs:
+  matrix_build:
+    strategy:
+      fail-fast: false
+      matrix:
+        # os: [ubuntu-latest, macos-latest]
+        # removed ubuntu build for now as it failed cause related to yeslogic-fontconfig-sys and fontique dlopen feature conflict
+        os: [macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v5
+      - name: Cache build
+        id: cache-build
+        uses: actions/cache@v4
+        with:
+          path: examples/servo/target/release/servo_example
+          key: cache-matrix-${{ matrix.os }}-build-${{ hashFiles('examples/servo/**') }}
+      - name: Install uv
+        if: steps.cache-build.outputs.cache-hit != 'true'
+        uses: astral-sh/setup-uv@v7
+      - name: Install rust
+        if: steps.cache-build.outputs.cache-hit != 'true'
+        uses: ./.github/actions/setup-rust
+      - name: Install dependencies
+        if: steps.cache-build.outputs.cache-hit != 'true'
+        uses: ./.github/actions/install-linux-dependencies
+      - name: Install skia dependencies
+        if: steps.cache-build.outputs.cache-hit != 'true'
+        uses: ./.github/actions/install-skia-dependencies
+      - name: Build
+        if: steps.cache-build.outputs.cache-hit != 'true'
+        working-directory: examples/servo
+        run: cargo build --release
+  android_build:
+    runs-on: macos-latest
+    env:
+      CARGO_PROFILE_RELEASE_OPT_LEVEL: s
+      CARGO_INCREMENTAL: false
+      CARGO_APK_RELEASE_KEYSTORE: /Users/runner/.android/release.keystore
+      CARGO_APK_RELEASE_KEYSTORE_PASSWORD: ${{ secrets.ANDROID_KEYSTORE_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v5
+      - name: Cache build
+        id: cache-android
+        uses: actions/cache@v4
+        with:
+          path: examples/servo/target/release/apk/servo_example_lib.apk
+          key: cache-android-build-${{ hashFiles('examples/servo/**') }}
```

Comment:
I suggest to remove this cache for the time being. We're notoriously low on cache space and unless this is "CI performance critical" the cache will not be populated anyway.